contracts/interchain-token/tests/test.rs

```diff
@@ -0,0 +1,86 @@
+#![cfg(test)]
+extern crate std;
+
+use axelar_soroban_std::{
+    assert_contract_err, assert_invoke_auth_err, assert_invoke_auth_ok, assert_last_emitted_event,
+};
+use interchain_token::contract::InterchainToken;
+use interchain_token::error::ContractError;
+use interchain_token::InterchainTokenClient;
+use soroban_sdk::testutils::{Address as _, MockAuth, MockAuthInvoke};
+use soroban_sdk::{Address, Env, IntoVal, Symbol};
+
+fn setup_env<'a>() -> (Env, Address, InterchainTokenClient<'a>) {
+    let env = Env::default();
+
+    let contract_id = env.register_contract(None, InterchainToken);
+    let client = InterchainTokenClient::new(&env, &contract_id);
+
+    (env, contract_id, client)
+}
+
+fn initialize(_env: &Env, client: &InterchainTokenClient, owner: Address) {
```

Comment:
Feels a little overkill to have a function for a single line (unless we expect to add more here eventually?)